### ISSUE TITLE
Trying to record audo, DOMException: Permission denied

### ISSUE THREAD

**Issue by spiffytech** [ionic-team/capacitor#861]
This is a follow-up to #802. Here's a tiny sample app I made with Vue and Capacitor 1.0.0-beta.8 that tries to open an audio source and play it back out the speaker.

```
<template>
  <div>
    <p>Hello, world!</p>
    <audio ref="audioOut" autoplay controls style="display: none;"></audio>
  </div>
</template>

<script>
export default {
  async mounted() {
    const stream = await navigator.mediaDevices.getUserMedia({audio: true, video: false});
    this.$refs.audioOut.srcObject = stream;
  },
}
</script>
```

I have also added `<uses-permission android:name="android.permission.RECORD_AUDIO" />` to `android/app/src/main/AndroidManifest.xml`

When I install this on my Phone (OnePlus 5, Android 8.1) I get no audio playback, and the Chrome USB debugger, I see `DOMException: Permission Denied`. I can retrigger than error by running `navigator.mediaDevices.getUserMedia({audio: true, video: false})` from the Chrome console.

**Comment by jcesarmobile** [comment]
Can you try also adding `<uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS"/>`?
As I pointed out on #802 it's also necessary for getting the audio recording. 
I have that one and can't reproduce.

**Comment by jcesarmobile** [comment]
No, that only request the permission, but it also needs to exist in the AndroidManifest.xml to work (and for Android 5 devices that don't request permission on run time but on install time).

We should probably add this 2 permission to the default template so it works out of the box.

**Comment by timmyrosen** [comment]
@jcesarmobile Dude, this saved my day. Been struggling with this for 5 hours now man, thank you!
